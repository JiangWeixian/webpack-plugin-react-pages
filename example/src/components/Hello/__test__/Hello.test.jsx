import React from 'react'
import renderer from 'react-test-renderer'
import Hello from '@/components/Hello/Hello'

describe('hello', () => {
  test('example jest with ts', () => {
    const component = renderer.create(<Hello />)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
