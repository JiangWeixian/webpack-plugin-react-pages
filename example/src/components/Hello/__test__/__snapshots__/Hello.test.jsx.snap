// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hello example jest with ts 1`] = `
<div
  className="hello"
/>
`;
